tests/conftest.py

```diff
@@ -359,7 +366,7 @@ def msg_template_factory(
     subject: str = "",
     stream_id: Optional[int] = None,
     recipients: Union[str, List[Dict[str, Any]]] = "PTEST",
-):
+) -> Message:
```

Comment:
We could, `conftest` already has scattered type annotations, and I can see how your suggestion provides more context on how this change affects all the templates being generated. The only thing that I'm a little unsure of is how that maintains that practice of optionally adding type annotations, albeit in a commit prior to adding type annotations completely.

Reply:
We could, `conftest` already has scattered type annotations, and I can see how your suggestion provides more context on how this change affects all the templates being generated. The only thing that I'm a little unsure of is how that maintains that practice of optionally adding type annotations, albeit in a commit prior to adding type annotations completely.

Reply:
I've accommodated this change, it makes sense now, although scattered, since annotations are being added soon after, and we have the benefit of additional context.

Reply:
I've accommodated this change, it makes sense now, although scattered, since annotations are being added soon after, and we have the benefit of additional context.